fastlane/Fastfile

```diff
@@ -0,0 +1,286 @@
+require 'yaml'
+require_relative 'version'
+
+desc "Commit version file"
+private_lane :github_commit_version do |options|
+  require 'openssl'
+  require 'jwt'
+  require 'octokit'
+  require 'base64'
+
+  branch = git_branch() # Commit to current branch
+  version_yml_path = options[:version_yml_path]
+  repo = options[:repo]
+  new_version_code = options[:new_version_code]
+  new_version_name = options[:new_version_name]
+  app_id = options[:app_id]
```

Comment:
Are these options given as "arguments" into the lane? Can you document them at the top of the lane?